.script/utils/dataConnectorCheckers/dataTypeChecker.ts

```diff
@@ -0,0 +1,20 @@
+import { DataType } from "../dataConnector";
+import { DataConnectorValidationError } from "../validationError";
+
+export function isValidDataType(dataTypes: Array<DataType>): boolean {   
```

Comment:
where is this function called?

Reply:
updated.
following are the CEF and Syslog data connector which is having spaces in Datatype.
https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/ThycoticSecretServer_CEF.json    
https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/WireXsystemsNFP(1b).json     - CEF
https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/CiscoISE/Connector_Cisco_ISE.json  - Syslog

Reply:
based on the example it seems that if there is a space inside the braces of the data type then it's ok. It seems we don't need to invalidate in that case. is that correct?

Reply:
Yes, We are checking there should not be spaces in data type of Rest API and Azure function data connector.